models/Intern.js

```diff
@@ -0,0 +1,57 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+const internsSchema = Schema({
+  name: {
+    type: String,
+    required: true
+  },
+  internId:{
+    type: Number,
+    required: true
+  },
+  email: {
+    type: String,
+    required: true,
+    unique: true
+  },
+  phoneNumber: {
+    type: Number,
+    required: true
+  },
+  cvLink: {
+    type: String,
+    required: true,
+    default: 'blank'
+  },
+  interest: {
+    type: String, 
+    required: true
+  },
+  country: {
+    type: String,
+    required: true
+  },
+  state: {
+    type: String
+  }, 
+  employmentStatus: {
+    type: String,
+    enum: ['employed', 'unemployed', 'student'],
+    default: 'unemployed'
+  },
+  about: {
+    type: String
+  },
+  registrationDate:{
```

Comment:
why this field

Reply:
according to HNGi7 format
<img width="1440" alt="Screenshot 2020-07-24 at 10 33 33 AM" src="https://user-images.githubusercontent.com/45487954/88454586-90ff8c80-ce25-11ea-9366-8d4aa4f1505d.png">